docs/dev/new_check_howto.md

```diff
@@ -88,187 +91,166 @@ A Check is a Python class with the following requirements:
         if not url or not search_string:
-            raise CheckException("Configuration error, please fix my_check.yaml")
+            raise ConfigurationError('Configuration error, please fix awesome.yaml')
 
         try:
-            r = requests.get(url)
-            r.raise_for_status()
-            if search_string in r.text:
-                # Page is accessible and the string is present.
-                self.service_check('my_check.all_good', self.OK)
-            else:
-                # Page is accessible but the string was not found.
-                self.service_check('my_check.all_good', self.WARNING)
+            response = requests.get(url)
+            response.raise_for_status()
+
+        # Something went horribly wrong
         except Exception as e:
-            # Something went horribly wrong. Ideally we'd be more specific...
-            self.service_check('my_check.all_good', self.CRITICAL, e)
+            # Ideally we'd use a more specific message...
+            self.service_check('awesome.search', self.CRITICAL, message=str(e))
+
+        # Page is accessible
+        else:
+
+            # search_string is present
+            if search_string in response.text:
+                self.service_check('awesome.search', self.OK)
+
+            # search_string was not found
+            else:
+                self.service_check('awesome.search', self.WARNING)
 ```
 
-To learn more about the base Python class, see the [Python API documentation][5]. Moving along, let's dive into tests, which are an important part of any project (and *required* for inclusion in `integrations-extras`).
+To learn more about the base Python class, see the [Python API documentation][5]. Moving along, let's dive into tests, which are an important part of any project (and _required_ for inclusion in `integrations-extras`).
 
 ### Writing tests
 
-There are two basic types of tests: unit tests for specific elements, and integration tests that execute the `check` method and verify proper metrics collection. Note that [pytest][6] and [tox][7] are used to run the tests.
+There are two basic types of tests: unit tests for specific functionality, and integration tests that execute the `check` method and verify proper metrics collection. Note that [pytest][6] and [tox][7] are used to run the tests.
 
 For more information, see the [Datadog Checks Dev documentation][8].
 
-The first part of the `check` method below retrieves two pieces of information we need from the configuration file. This is a good candidate for a unit test. Open the file at `my_check/tests/test_check.py` and replace the contents with something like this:
+The first part of the `check` method we wrote retrieves and verifies two pieces of information we need from the configuration file. This is a good candidate for a unit test. Open the file at `awesome/tests/test_awesome.py` and replace the contents with something like this:
```

Comment:
```suggestion
The first part of this `check` method retrieves and verifies two pieces of information we need from the configuration file. This is a good candidate for a unit test. Open the file at `awesome/tests/test_awesome.py` and replace the contents with something like this:
```

Reply:
Since the method was defined way up above are you sure `this` is correct?

Reply:
I see what you mean. I changed my suggestion back to `the`